snap/snapcraft.yaml

```diff
@@ -194,12 +195,12 @@ parts:
                     --ta-version 0 \\
                     --in \$SNAP/tmp/unsigned_elf/"\$user_ta_uuid".stripped.elf --out \$SNAP_COMMON/lib/optee_armtz/"\$user_ta_uuid".ta
                 done
-            elif [ "\$1" == "unsign-ta" ]
+            elif [ "\$cmd" == "unsign-ta" ]
             then
                 cp \$SNAP/tmp/lib/optee_armtz/* \$SNAP_COMMON/lib/optee_armtz
-            elif [ "\$1" == "import" ]
+            elif [ "\$cmd" == "import" ]
             then
-                cp \$2/* \$SNAP_COMMON/lib/optee_armtz
+                cp \$@/* \$SNAP_COMMON/lib/optee_armtz
```

Comment:
Need to use `$1` here, otherwise it could fail if multiple parameters are given.

One further question is should we check the number of parameters here?

Reply:
I suppose we don't have to worry about multiple parameters in this case. Since the execution only copies the file, if it receives the wrong parameter (e.g., multiple parameters), the `cp` command will raise an error, and nothing will go wrong.

Reply:
I don't have strong opinion on this. Let @kiya956 decide